Tests/conf.json

```diff
@@ -2816,6 +2816,7 @@
     },
     "skipped_integrations": {
         "_comment1": "~~~ NO INSTANCE ~~~",
+        "ArcSight Logger": "Issue 24303",
```

Comment:
are you synced with @ShahafBenYakir on this ?

Reply:
Yup